document/footnotes.go

```diff
@@ -0,0 +1,81 @@
+package document
```

Comment:
https://github.com/unidoc/unioffice/wiki/UniOffice-Developer-Guide#7-license-header-in-each-go-file